src/docs/index.rst

```diff
@@ -249,11 +249,5 @@ LLNL-CODE-805541
    :hidden:
    :maxdepth: 2
 
-   sphinx/developer_guide
-   sphinx/style_guide
-   sphinx/logging
-   sphinx/expr_templates
-   sphinx/memory_checking
-   sphinx/profiling
-   sphinx/docker_info
-   sphinx/input_schema
+   sphinx/user_guide/index
```

Comment:
We should probably change the code reference to Smith at the top of this file. Also, do we want the build instructions to be on a separate page?

Reply:
I think having the build instructions/quickstart as a separate page makes sense